lib/stripe/stripe_client.rb

```diff
@@ -148,28 +148,14 @@ def execute_request(method, path,
 
       body_params = nil
       query_params = nil
-      case method.to_s.downcase.to_sym
+      case method
```

Comment:
We added a `raise ArgumentError` above that fires if `method` isn't a symbol, so we can remove this hedge.